verification/envs/python_3/main.py

```diff
@@ -47,63 +32,67 @@ class PlayerRefereeClient(RefereeClient):
     def __init__(self, *args, **kwargs):
         super().__init__(*args, **kwargs)
         self._events = {}
+        self.events_call = Queue()
         self.runner = None
 
     def set_runner(self, runner):
         self.runner = runner
 
-    def _get_response_json(self):
-        response = super()._get_response_json()
-        if not response:
-            return
+    def _request(self, data, skipp_result=None):
+        data['status'] = 'success'
+        return self.request(data, skipp_result, skip_clean_up=True)
 
-        method = response.get('method')
-        if not method:
-            return response
+    def request(self, *args, **kwargs):
+        skip_clean_up = kwargs.pop('skip_clean_up', None)
```

Comment:
what if 
`def request(self, *args, skip_clean_up = None, **kwargs):`
and remove line 46